packages/lesswrong/components/seasonal/HomepageMap/HomepageCommunityMap.tsx

```diff
@@ -134,15 +134,16 @@ const LocalEventMapMarkerWrappersComponent = registerComponent("LocalEventMapMar
 });
 
 
-export const HomepageCommunityMap = ({classes}: {
+export const HomepageCommunityMap = ({requestUserLocation = true, classes}: {
+  requestUserLocation?: boolean,
   classes: ClassesType,
 }) => {
   const { LocalEventMapMarkerWrappers, HomepageMapFilter } = Components
 
   const currentUser = useCurrentUser()
  
   // this is unused in this component, but for Meetup Month it seems good to force the prompt to enter location.
-  useUserLocation(currentUser)
+  useUserLocation(currentUser, !requestUserLocation)
```

Comment:
The second parameter here is an (optional) `dontAsk?: boolean` which controls whether we use the browser's geolocation API to request the user's location.